docs/tutorials/geodocker-geomesa/index.rst

```diff
@@ -0,0 +1,14 @@
+GeoDocker GeoMesa
+===================
+
+The GeoDocker-GeoMesa project makes standing up a GeoMesa cluster
+running on Accumulo very easy. Currently we support running a
+GeoDocker-GeoMesa cluster locally, on a single machine, and on an
```

Comment:
slightly ambiguous - could we change to parens maybe? 'locally (on a single machine), or on an AWS cluster'